default.nix

```diff
@@ -168,16 +168,16 @@ rec {
       ln -s ${compressedJs frontend} $out/frontend.jsexe
     ''; #TODO: run frontend.jsexe through the asset processing pipeline
 
-  server = exe: hostName:
+  server = exe: hostName: sslHost: adminEmail:
     let system = "x86_64-linux";
         nixos = import (pkgs.path + /nixos);
         https = (import lib/https {}).module {
           backendPort = 8000; # TODO read from config
-          # sslConfig = {
-          #   hostName = "example.com";
-          #   adminEmail = "webmaster@example.com";
-          #   subdomains = [ "www" ];
-          # };
+          sslConfig = {
+            hostName = sslHost;
+            adminEmail = adminEmail;
+            subdomains = [ ];
+          };
```

Comment:
It looks like this makes HTTPS *mandatory*.  It should not be mandatory - HTTPS termination doesn't make sense, e.g., if the server is running behind a load balancer.

Reply:
Pushed a change that includes `--disable-https` to disable the by-default https behaviour.

Reply:
We should not use "" to denote a missing sslHost; it should be `null`.

Reply:
Done.